<launch>
    <!-- joy node -->
    <node respawn="true" pkg="joy" type="joy_node" name="joy_node">
        <param name="dev" type="string" value="/dev/input/js0" />
        <!--重复发送手柄话题-->
        <param name="autorepeat_rate" type="double" value="10" />
        <param name="deadzone" value="0.12" />
    </node>

    <param name="axis_linear" value="1" type="double"/>
    <param name="axis_angular" value="0" type="double"/>
    <param name="button_emergency" value="0" type="int"/>
    <param name="button_gear_up" value="5" type="int"/>
    <param name="button_gear_down" value="4" type="int"/>
    <param name="max_linear_speed" value="0.2" type="double"/>
    <param name="max_angular_speed" value="0.2" type="double"/>
    <node pkg="joy_control" type="joy_control_node" name="joy_control_node"/>
</launch>
   